### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by michaelmang**
Adds some basic documentation for the package.

When we open up outside contribution, we can update the readme accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,196 @@
...
+  - [❓ Questions](#-questions)
+- [Contributors](#contributors)
+
+## The problem
+
+You want to utilize the Helix UI webcomponents for your React application.
+As a part of this goal, you have to write and maintain React components
+that wrap the webcomponents with required boilerplate. You might want to
+use a tool like [Storybook](https://storybook.js.org/) to see what these
+wrappers look like.
```
Comment (nicko-winner) [inline README.md:39]: Another idea / approach.
```suggestion
You ❤️[Helix UI webcomponents library](https://helixdesignsystem.github.io/helix-ui/) and want to use them in your React application. You then realized React support for webcomponents is [underwelming](https://custom-elements-everywhere.com/) and requires you to write React compatibility wrappers for each web component that uses events. 
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,196 @@
...
+  id: PropTypes.string,
+  size: PropTypes.oneOf(SIZES),
+  children: PropTypes.node.isRequired,
+  className: PropTypes.string,
+  open: PropTypes.bool,
+  onClose: PropTypes.func,
+  onOpen: PropTypes.func,
+};
+
+export default Modal;
```
Comment (nicko-winner) [inline README.md:118]:
```suggestion
<hxDisclosure
  ariaControls="demo-modal"
  className="hxBtn"
>
  Open Modal
</hxDisclosure>

<hxModal id="demo-modal">
  <header>
    ...
  </header>
  <hxDiv>
    ...
  </hx-div>
  <footer>
    <Button variant="primary">Confirm</Button>
    <Button>Cancel</Button>
  </footer>
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,196 @@
...
+## 📚 Storybook
+[Storybook](https://storybook.js.org/) is an open source tool for developing UI components
+in isolation for React, Vue, and Angular.
+You can use this as a playground for all our components.
+
+1. Install Yarn
+
+```bash
+npm i -g yarn
+```
```
Comment (nicko-winner) [inline README.md:143]: We might not need to tell them how to install yarn. They should be able to do `npm install` or `yarn install` either will work if they have the tooling.  (for contributors we can talk more about only using yarn)

**Comment by michaelmang** [comment]
@nicko-winner feedback addressed, ready for a re-review

File: README.md [hunk 4]
```diff
@@ -1,3 +1,173 @@
...
+a 👍. This helps maintainers prioritize what to work on.
+
+[**See Feature Requests**](https://github.com/HelixDesignSystem/helix-react/issues?q=is%3Aopen+is%3Aissue+label%3Aenhancement)
+
+### ❓ Questions
+
+For questions related to using the library, please visit a support community
+instead of filing an issue on GitHub.
+
+[Slack](https://join.slack.com/share/zt-edajgkat-LG_5SXYoCeIzknrQPVWaTA)
```
Comment (nicko-winner) [inline README.md:161]: is there a policy around making a publicly accessible channel? or only rackers can access this channel?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,173 @@
...
+```bash
+yarn install
+```
+
+2. Run Storybook
+
+```bash
+npm start
+```
+The script should automatically open Storybook in your browser, but you can always navigate to http:http://localhost:6006/?path=/story
```
Comment (100stacks) [inline README.md:133]: Please remove the extra `http:`.

File: README.md (hunk 4 above)
Comment (100stacks) [reply README.md:161]: You bring up a good point about a publicly accessible channel.

External contributors will have to submit an issue.  We can make an announcement internally on Slack when we're ready to go live.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,173 @@
...
+[Slack](https://join.slack.com/share/zt-edajgkat-LG_5SXYoCeIzknrQPVWaTA)
+
+## Contributors
+
+Thanks goes to these people:
+
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/michaelmang"><img src="https://avatars0.githubusercontent.com/u/10120600?s=400&u=7a37b1eba79b6ee2c7753a457b8a33998595fe20&v=4" width="100px;" alt=""/><br /><sub><b>James Thompson</b></sub></a><br /><a href="https://github.com/HelixDesignSystem/helix-react/commits?author=100Stacks" title="Commits">💻</a></td>
+    <td align="center"><a href="https://github.com/michaelmang"><img src="https://avatars0.githubusercontent.com/u/35456401?s=400&v=4" width="100px;" alt=""/><br /><sub><b>Nicko Winner-Arroyo</b></sub></a><br /><a href="https://github.com/HelixDesignSystem/helix-react/commits?author=nicko-winner" title="Commits">💻</a></td>
```
Comment (100stacks) [inline README.md:170]: Please update the link:

```html
<a href="https://github.com/nicko-winner">
```

File: README.md [hunk 7]
```diff
@@ -1,3 +1,173 @@
...
+
+[Slack](https://join.slack.com/share/zt-edajgkat-LG_5SXYoCeIzknrQPVWaTA)
+
+## Contributors
+
+Thanks goes to these people:
+
+<table>
+  <tr>
+    <td align="center"><a href="https://github.com/michaelmang"><img src="https://avatars0.githubusercontent.com/u/10120600?s=400&u=7a37b1eba79b6ee2c7753a457b8a33998595fe20&v=4" width="100px;" alt=""/><br /><sub><b>James Thompson</b></sub></a><br /><a href="https://github.com/HelixDesignSystem/helix-react/commits?author=100Stacks" title="Commits">💻</a></td>
```
Comment (100stacks) [inline README.md:169]: Please update the link:

```html
<a href="https://github.com/100stacks">
```

File: README.md [hunk 8]
```diff
@@ -1,3 +1,173 @@
...
+
+## Guiding Principles
+
+We try to only expose basic wrappers of Helix UI webcomponents.
+Each wrapper corresponds with a component listed on the [Helix UI documentation](https://helixdesignsystem.github.io/helix-ui/):
+
+![helix-ui-documentation](https://user-images.githubusercontent.com/22566333/82083617-07f70700-96b8-11ea-9581-da1796737e52.png)
+
+These components are just the "building blocks" of other Helix patterns listed in
+the [Helix design documentation](http://helix.rax.io/status.html).
```
Comment (100stacks) [inline README.md:112]: That list is a little dated.  Let's not link to it please.

**Comment by michaelmang** [comment]
@100stacks this is ready for re-review

**Pull request merged**